src/util/apis/generated/models/activeDeviceCountsMonthlyItem.js

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Why so many new files? Did something change in the generator or did these actually appear since we last ran it?

Reply:
It's like it added files for all the parameters of all API endpoints.

Reply:
Dunno why but our swagger json is just much larger and more detailed than it used to be.
It indeed contains the details for all parameters & response types. I can't find what changed that causes this, but it looks fine to me.

We may want to consider shrinking the size of the CLI package by filtering out only the files we need at runtime, but that's my only concern here.